### PR TITLE
Invite the welcome user after registration if configured

### PR DESCRIPTION
**Pull request by lukebarnard1**
This will shift focus to the welcome user DM.

We probably don't want to do this for teams, but I shall leave that for another PR that fixes teams WRT to new-guest-access.

### CODE REVIEW

File: src/components/structures/MatrixChat.js [hunk 1]
```diff
@@ -578,7 +578,7 @@ module.exports = React.createClass({
                 this.setState({loggingIn: true});
                 break;
             case 'on_logged_in':
-                this._onLoggedIn(payload.teamToken, payload.isPasswordStored);
+                this._onLoggedIn(payload.teamToken);
```
Comment (dbkr) [inline src/components/structures/MatrixChat.js]: Was this intentional? It doesn't look related.

Comment (lukebarnard1) [inline src/components/structures/MatrixChat.js]: Intentional, yes. Related not really - it'll make a bit more sense when #881 lands (because that will get rid of `isPasswordStored`
